DmitriySkakun/HW4/src/app/components/Blog.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Post from './Post';
+
+class Blog extends React.Component{
+    render(){
+        const items = this.props.items.map((item, index) => {
+            return <Post key = {index}>{item.title}{item.postBody}{item.author}</Post>
```

Comment:
```suggestion
            return <Post key = {index} {...item}/>
```
Чем все в children передавать, лучше отдельные props'ы передавать;)